src/app/api/gql/fragments/fragment-dataset-metadata-data.graphql

```diff
@@ -0,0 +1,20 @@
+fragment DatasetMetadataData on Dataset {
+  metadata {
+    currentInfo {
+      ...DatasetCurrentInfo
+    }
+    currentLicense {
+      ...License
+    }
+    currentWatermark
+    ...DatasetCurrentTransform
```

Comment:
Would it be better instead of:
```gql
...DatasetCurrentTransform
```
to have:
```gql
currentTransform {
   ...DatasetTransform
}
```
This will make `DatasetTransform` a more reusable fragment (as it can be used to represent any transformation block, not only the current one). It will also replace both `DatasetCurrentTransform` and `DatasetTransformContent`.